apps/publish/enqueue/enqueue_service.py

```diff
@@ -212,6 +212,9 @@ def publish(self, doc, target_media_type=None):
             logger.exception('Nothing is saved to publish queue for story: {} for action: {}'.
                              format(doc[config.ID_FIELD], self.publish_type))
 
+        # publish to content api
+        get_resource_service('content_api').publish(doc, subscribers)
```

Comment:
If we don't load the content-api module in our settings we can turn this off initially I guess. Is that right?